boxfile.yml

```diff
@@ -105,20 +113,25 @@ worker.sidekiq:
     data.storage:
       - public/system
 
-  cron:
-    - id: generate_static_gifs
-      schedule: '*/15 * * * *'
-      command: 'bundle exec rake mastodon:maintenance:add_static_avatars'
 
+worker.cron_only:
+  start: sleep 365d
+
+  writable_dirs:
+    - tmp
+
+  log_watch:
+    rake: 'log/production.log'
+
+  network_dirs:
+    data.storage:
+      - public/system
+
+  cron:
     - id: update_counter_caches
```

Comment:
Remove this one

Reply:
This is done automatically as well, then? Or do the docs just not explain its purpose very well?

Reply:
This CLI command has been used in the upgrade when counter caches were first introduced and needed to be filled.

Reply:
Ok, cool. Consider it gone.